src/common/Breadcrumbs/breadcrums.scss

```
@@ -1,32 +1,47 @@
-@import "../../scss/mixins";
+@import '../../scss/mixins';
 @import '../../scss/colors';
 
 .breadcrumbs {
+  @include resetSpaces;
+
   font-size: 20px;
   line-height: 23px;
-  @include resetSpaces;
+
   &__list {
     @include resetSpaces;
-    list-style-type: none;
+
     display: flex;
     flex-wrap: wrap;
     align-items: center;
     font-weight: 400;
     letter-spacing: 0.00938em;
+    list-style-type: none;
   }
+
   &__item {
+    position: relative;
     color: $topaz;
+
     &:last-child {
       color: $mulledWine;
       font-weight: 500;
       font-size: 20px;
       line-height: 23px;
     }
   }
-  &_separator {
+
+  &__separator {
     display: flex;
     margin: 0 8px;
-    user-select: none;
     color: $topaz;
+    user-select: none;
+
```

Let's add a nice transition to rotation:
```suggestion
    transition: transform .2s linear;

```